docs/en/user_guides/useful_tools.md

```diff
@@ -32,30 +32,30 @@ Params: 1.52 M
 (2) Some operators are not counted into FLOPs like GN and custom operators.
 You can add support for new operators by modifying [`mmcv/cnn/utils/flops_counter.py`](https://github.com/open-mmlab/mmcv/blob/master/mmcv/cnn/utils/flops_counter.py).
 
-### Publish a model
+## Publish a model
 
 Before you upload a model to AWS, you may want to
 (1) convert model weights to CPU tensors, (2) delete the optimizer states and
 (3) compute the hash of the checkpoint file and append the hash id to the filename.
 
 ```shell
-python tools/publish_model.py ${INPUT_FILENAME} ${OUTPUT_FILENAME}
+python tools/model_converters/publish_model.py ${INPUT_FILENAME} ${OUTPUT_FILENAME}
 ```
 
 E.g.,
 
 ```shell
-python tools/publish_model.py work_dirs/example_exp/latest.pth example_model_20200202.pth
+python tools/model_converters/publish_model.py work_dirs/example_exp/latest.pth example_model_20200202.pth
```

Comment:
maybe we should give time tag by script. Example: https://github.com/open-mmlab/mmgeneration/blob/master/tools/publish_model.py#L29.

Reply:
> maybe we should give time tag by script. Example: https://github.com/open-mmlab/mmgeneration/blob/master/tools/publish_model.py#L29.

to be done in the other PR. 
In this PR, we only discuss docs.